robot.py

```diff
@@ -110,8 +108,6 @@ def createObjects(self):
 
         self.driverStation = DriverStation.getInstance()
 
-        # Make sure default autonomous is big brain
```

Comment:
How was this tested?

Reply:
In sim, I didn't think to test it further than looking at the network tables
I should've checked states during autonomous